<launch>
	<node pkg="asl_turtlebot" type="goal_supervisor.py" name="goal_supervisor" output="screen"/>

	<node pkg="asl_turtlebot" type="food_localizer.py" name="food_localizer" output="screen"/>

	<node pkg="asl_turtlebot" type="display_food_location.py" name="display_food" output="screen"/>

	<node pkg="asl_turtlebot" type="display_goal_location.py" name="display_goal" output="screen"/>

	<node pkg="asl_turtlebot" type="goal_commander.py" name="goal_commander" output="screen"/>

	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find asl_turtlebot)/launch/project_rviz.rviz" />

</launch>
